scrapers/add_district_db_entry.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+import sys
+import sqlite3
+import traceback
+import os
+
+import scrape_common as sc
+
+__location__ = os.path.realpath(
+    os.path.join(
+        os.getcwd(),
+        os.path.dirname(__file__)
+    )
+)
+
+input_failures = 0
+
+try:
+    DATABASE_NAME = os.path.join(__location__, 'data.sqlite')
+    conn = sqlite3.connect(DATABASE_NAME)
+
+    i = 0
+    for line in sys.stdin:
+        dd = sc.DistrictData()
+        if dd.parse(line.strip()):
+            c = conn.cursor()
+            try:
+                print(dd)
+
+                c.execute(
+                    '''
+                    INSERT INTO data (
+                      DistrictId,
+                      District,
+                      Canton,
+                      Date,
+                      Week,
+                      Year,
+                      Population,
+                      TotalConfCases,
+                      NewConfCases,
+                      TotalDeaths,
+                      NewDeaths,
+                      SourceUrl
+                    )
+                    VALUES
+                    (?,?,?,?,?,?,?,?,?,?,?,?)
+                    ON CONFLICT (DistrictId, District, Canton, Date, Week, Year)
+                    DO UPDATE SET
```

Comment:
Unfortunetely this does only work in newer versions of sqlite. I tried to use this UPSERT logic for the other scraper as well but had to give up, since it's a hassle to get the latest sqlite version on GitHub Actions.

Instead I opted for a version that works with all versions of sqlite by trying to INSERT, check if the unique constraints was violated (`except sqlite3.IntegrityError:`), and if so, use an UPDATE statement instead. It's not beautiful, but it works.

I will merge this PR and create a follow-up with the new INSERT/UPDATE logic.